dp/generate_climos.py

```diff
@@ -134,19 +134,34 @@ def climo_outputs(time_resolution):
     # Apply per-variable processing
     # - Scale pr variable if it is not in desired units
     #   - TODO: Use UDUNITS here for more robust processing of units? It has a fairly heavy API. But this is uuuugly.
```

Comment:
You may want to check out the [``pint`` library](https://pint.readthedocs.io/en/0.7.2/) for units handling.